### PR TITLE
Add a script to synchronise translations from the English file to the …

### PR DESCRIPTION
**Pull request by LEDfan**
…local files

@RobinJadoul what do you think?

If found useful I can add a CI job.

### CODE REVIEW

**Comment by LEDfan** [comment]
E.g. the output on #71:
```
[zh_CN] Added              initSSFailed
[zh_CN] Needs translation  initSSFailed
```

File: sync_translation.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  with open(target_file_name, 'w') as file:
+    json.dump(target, file, indent=2, ensure_ascii=False)
+
+  return {"added": added, "updated": updated, "nottranslated": nottranslated}
+
+
+if __name__ == "__main__":
+
+  with open(os.path.join('_locales', 'en', 'messages.json')) as file:
```
Comment (RobinJadoul) [inline sync_translation.py]: maybe use `__file__` to make sure the current working directory doesn't interfere

File: sync_translation.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    enTranslation = json.load(file)
+
+  for f in os.listdir('_locales'):
+    if f != "en":
+      stats = synctranslation(enTranslation, os.path.join("_locales", f, "messages.json"))
+
+      if len(stats["added"]) > 0:
```
Comment (RobinJadoul) [inline sync_translation.py]: This check isn't needed, since the `for` wouldn't loop otherwise.

**Comment by LEDfan** [comment]
@RobinJadoul done :smile:

File: sync_translation.py [hunk 3]
```diff
@@ -1,59 +1,78 @@
...
-      target[id] = base[id]
-    elif target[id]["description"] != base[id]["description"]:
-      updated.append(id)
-      target[id]["description"] = base[id]["description"]
+        if target[id]["message"] == base[id]["message"]:
+            not_translated.append(id)
 
-    if target[id]["message"] == base[id]["message"]:
-      nottranslated.append(id)
+    print(join(base_path, target_file_name))
```
Comment (RobinJadoul) [inline sync_translation.py]: Too much output this way?

File: sync_translation.py [hunk 4]
```diff
@@ -1,59 +1,78 @@
...
+#
+# Keywi is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# You should have received a copy of the GNU General Public License
+# along with Keywi.  If not, see <http://www.gnu.org/licenses/>.
+
+
 # This script synchronises a translated language file with the English one
```
Comment (RobinJadoul) [inline sync_translation.py]: This could actually be a docstring, but I guess a comment is fine as well

**Pull request merged**